metadata-ingestion/tests/integration/powerbi/golden_test_container.json

```diff
@@ -65,14 +65,48 @@
         "runId": "powerbi-test"
     }
 },
+{
```

Comment:
same here, user should not get added in existing golden files, looks like your owner related changes are breaking previous configuration

Reply:
@aezomz looks like this one is still unresolved - I think this is the last pending item on this one.

Reply:
Same thing, i checked. the json is being re-ordered. I am not sure why. 

https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/tests/integration/powerbi/golden_test_container.json#L502